Framework/Core/Windows/Devices/Pwm/PwmController.cs

```diff
@@ -0,0 +1,86 @@
+using System;
+using Windows.Devices.Pwm.Provider;
+
+namespace Windows.Devices.Pwm
+{
+    public sealed class PwmController
+    {
+        private IPwmControllerProvider m_provider;
+        private double m_minFrequency;
+        private double m_maxFrequency;

```

Comment:
No, remove the cacheing, since these are not expected to change then they aren't needed very often either so the performance overhead of another call to native code isn't as big an issue as storing the data twice. (RAM s a VERY limited resource on these micro controllers so when it comes to tradeoffs the general rule is favor reduced memory usage unless tere is a measured wuantifiable justification otherwise)